internal/operator/cluster/upgrade.go

```diff
@@ -51,6 +52,24 @@ const (
 	postgresGISHAImage = "crunchy-postgres-gis-ha"
 )
 
+// nssWrapperForceCommand is the string that should be appended to the sshd_config file as
+// needed for nss_wrapper support when upgrading from versions prior to v4.7
+const nssWrapperForceCommand = `# ensure nss_wrapper env vars are set when executing commands as needed for OpenShift compatibility
+ForceCommand NSS_WRAPPER_SUBDIR=ssh . /opt/crunchy/bin/nss_wrapper_env.sh && $SSH_ORIGINAL_COMMAND`
+
+// the following regex expressions are used when upgrading the sshd_config file for a PG cluster
+var (
+	// nssWrapperRegex is the regular expression that is utilized to determine if the nss_wrapper
+	// ForceCommand setting is missing from the sshd_config (as it would be for versions prior to
+	// v4.7)
+	nssWrapperRegex = regexp.MustCompile(nssWrapperForceCommand)
+
+	// nssWrapperRegex is the regular expression that is utilized to determine if the UsePAM
+	// setting is set to 'yes' in the sshd_config (as it might be for versions up to v4.6.1,
+	// v4.5.2 and v4.4.2)
+	usePAMRegex = regexp.MustCompile(`(?m)^UsePAM\s*yes`)
```

Comment:
@andrewlecuyer I would make the `yes` case insensitive

Reply:
OK, though that would mean we're essentially fixing a reconfigured OpenSSH server, since OpenSSH requires these to be lowercase (i.e. the only reason we'd see anything other than `yes` would be a mis-configuration).

Reply:
I believe `Yes` worked in EL7. So I'm OK with this.

Reply:
Made the entire thing case-insensitive after reading this in the `sshd` manual:

> note that keywords are case-insensitive and arguments are case-sensitive